### PR TITLE
Personalise2

### PR DESCRIPTION
**Pull request by benjaminrshill**
Sorry for the extra work!
Added some meta tags in the head and changed the contact section considerably (and probably some other stuff I can't remember)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,19 +169,29 @@ <h3>About</h3>
...
-		<section>
-			<a href="mailto:benjaminrshill@hotmail.com">
-				<img alt="email" src="images/email.png" />
+
+		<label for="contactCheck" class="contactLabel">
+		    <span class="contactButton">
+		        <img src="images/heart.png" alt="Contact" />
+			</span>
+		</label>
+		<input id="contactCheck" type="checkbox" />
```
Comment (snack-attack) [inline index.html:181]: I take it you're using inputs this way so you can track state and use transitions to create the drop downs? It's a bit unusual (but it works, which is great!). Once you get to use JS, you'll might want to refactor into more semantic elements here (as it's not actually an input, so accessibility-wise, this isn't ideal 😄 )

File: index.html [hunk 2]
```diff
@@ -167,19 +169,29 @@ <h3>About</h3>
...
-	<div class="contact">
+    <section class="contact">
 
 		<h3>Contact</h3>
-		<section>
-			<a href="mailto:benjaminrshill@hotmail.com">
-				<img alt="email" src="images/email.png" />
+
+		<label for="contactCheck" class="contactLabel">
+		    <span class="contactButton">
```
Comment (snack-attack) [inline index.html:177]: Any reason these spans can't just be regular old <div>s or <p>s or whatever? Just to make them a bit more semantic

**Review by snack-attack — COMMENTED** [review]
It's looking great Ben! Well done getting all this interactivity done in just CSS 👍

File: index.html (hunk 1 above)
Comment (benjaminrshill) [reply index.html:181]: Yes it's a bit of a pain knowing I'll have to go through and change a bunch! But really cool to have got it working just using selectors.

Have left a couple of reminder comments so hopefully nothing gets lost.

File: index.html (hunk 2 above)
Comment (benjaminrshill) [reply index.html:177]: The HTML validator wouldn't let me put anything except spans inside the label tag...!

**Pull request closed without merging**